.github/setup/global_resolver/action.yml

```diff
@@ -0,0 +1,96 @@
+name: Setup Akka resolver
+description: Adds Akka resolver globally for sbt and Maven
+runs:
+  using: "composite"
+  steps:
+    - name: Add Akka resolver for sbt
+      shell: bash
+      run: |
+        mkdir -p ~/.sbt/1.0
+        echo 'resolvers += "Akka library repository".at("https://repo.akka.io/maven/github_actions")' >> ~/.sbt/1.0/resolvers.sbt
+
+    - name: Add Akka Resolver to Scripted Tests
```

Comment:
Does anybody have any thoughts on how we can make this better for internal testing? We have pointed out in the CONTRIBUTING doc that devs need a global settings and sbt file, but those don't translate to sbt-test. Some thoughts:

* Add sbt-plugin/src/sbt-test/sbt-kalix/*/global/* to .gitignore and tell developers to  add the resolvers.sbt to each test global dir
* Add a custom task which `scripted` depends on that takes your `fullResolvers` and creates the files for you (probably still need the gitignore otherwise we need a post-task cleanup also)